README.md

```diff
@@ -1,2 +1,140 @@
+4. Development process review
+5. Develop roadmap
+
+
+
+## Planned deliverables
+
+We will work collaboratively to conduct research, synthesize collected data, and make findings and recommendations for Air Force to achieve a modularized version of AFRISS-TF that better meets user needs.
+
+The engagement focuses on identifying the key capabilities and characteristics of the future state system and outlining a roadmap for modernization that results in decreased training costs, increased accessibility and mobility, and greater flexibility through a more modular configuration. 
+
+## Risks and mitigations
+
+**Risk:** If it takes time to identify/schedule research participants, this could reduce our potential impact.  
+**Mitigation:** We press hard for contacts/interviews but move forward with other work in the meantime.
+
+**Risk:** Time is, by design, short. This engagement can go many ways as we define our eventual scope.  
+**Mitigation:** Team to commit to a path early on and dig in. Partner accepts that the work done will not be a comprehensive review of everything that is possible.
+
+**Risk:** Important concerns go unaddressed until they swell into larger problems.  
+**Mitigation:** Surface obstacles and anxieties early so we can address them as they arise.
+
+**Risk:** Balancing/prioritizing needs of three different components (Air Force, Reserve, Guard) could result in us either not targeting the highest-value opportunities or possibly spreading our efforts unnecessarily thin.  
+**Mitigation:** Determine when the time is right to focus on just one component and when it is best to include all three.
+
+**Risk:** If recommendations are not grounded in technical reality, roadmap delivered will not be actionable.  
+**Mitigation:** Bring on 18F engineer to support team. Partner continue to help with access to AFRS and vendor teams.
+
+
+## Tools
+ - [Team notes document](https://docs.google.com/document/d/1t5LBQJKpPr0poo91VDw96UE4qhISYygmZ2OyBWXNEZA/edit#)
+ - [Statement of Work](https://docs.google.com/document/d/1sRO7MsKep3qxmNSQkCK7-LgwH4ukavb6PasWjU-aCJQ/edit)
+ - [Google Drive folder](https://drive.google.com/drive/u/0/folders/15_hzcako6YoYnSl_0apZHuLhLVpLTLkS) 
+ - [Trello board](https://trello.com/b/1LRQgbGk/usaf-recruiting-service-path-analysis)
+
+
+## Tock and staffing
+
+- The 18F Tock code for billing time to this project is AFRS Path Analysis #784
+- [18F staffing issue](https://github.com/18F/staffing-and-resources/issues/481)
+
+## Project team and roles
+
+### AFRS
+
+**Lieutenant Colonel Gralyn Wimberly (Product Owner)** — Serve as single point of contact for AFRS for purposes of decisions needed to move the engagement forward. Collaborate with the development team. Keep team and stakeholders informed, soliciting their feedback. Highlight and work to mitigate risks. Communicate feedback and help prioritize the team’s effort. Assist with research recruitment, data collection, and analysis.
```

Comment:
Sp: should be Lieutenant Colonel `Grayln` Wimberly rather than `Gralyn`.

Reply:
My goodness, good catch.